src/main/kotlin/pl/ludek/poolserver/plugins/Routing.kt

```diff
@@ -11,6 +11,9 @@ fun Application.configureRouting() {
         get("/pool-info"){
             return@get call.respond(poolInfoController.answerServer())
         }
+        get("/relay-state"){
+           return@get call.respond(simpleModbusRTURelay.getStateAllRelay())
+        }
         route("/relay"){
```

Comment:
Please try to use the [path parameters](https://ktor.io/docs/requests.html#path_parameters) to avoid code duplication.
I would guess you should be able to do something like
```
route("/relay") {
    post("/{relay}/{state}") {
    return@post if (call.parameters["state"]?.toInt() == 1) {
        call.respond(simpleModbusRTURelay.onRelay(call.parameters["relay"] ))
    } esle {
        call.respond(simpleModbusRTURelay.offRelay(call.parameters["relay"] ))
    }
}
```